service/process.ml

```diff
@@ -8,16 +8,10 @@ let pp_cmd f = function
   | "", args -> pp_args f args
   | bin, args -> Fmt.pf f "(%S, %a)" bin pp_args args
 
-let pp_signal f x =
-  let open Sys in
-  if x = sigkill then Fmt.string f "kill"
-  else if x = sigterm then Fmt.string f "term"
-  else Fmt.int f x
-
 let pp_status f = function
-  | Unix.WEXITED x -> Fmt.pf f "exited with status %d" x
-  | Unix.WSIGNALED x -> Fmt.pf f "failed with signal %d" x
-  | Unix.WSTOPPED x -> Fmt.pf f "stopped with signal %d" x
+  | Unix.WEXITED x -> Fmt.pf f "exited with status %a" Fmt.Dump.signal x
```

Comment:
That is wrong, the exit code is not a signal number.

Reply:
Yeah you right.